Sources/XcodeGenKit/InfoPlistGenerator.swift

```diff
@@ -33,8 +33,9 @@ public class InfoPlistGenerator {
             targetInfoPlist["CFBundlePackageType"] = "FMWK"
         case .bundle:
             targetInfoPlist["CFBundlePackageType"] = "BNDL"
-        case .xpcService:
-            targetInfoPlist["CFBundlePackageType"] = "XPC"
+        case .xpcService,
+						 .appExtension:
```

Comment:
nitpick: indenting

Reply:
Fixed.

Can't wait for better `swift package generate-xcodeproj` that supports `usesTabs: false` 😄